src/resolvers.ts

```diff
@@ -36,13 +36,29 @@ export const resolvers = {
     async Users(_: any, args: any, context: any) {
       const userRepository = getConnection().getRepository(User);
       const reqToken: string = context.headers.authorization;
+      let hasPreviousPage: Boolean = false;
+      let hasNextPage: Boolean = false;
```

Comment:
alterar esses tipos para os minúsculos, [documentação aqui](https://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html#number-string-boolean-symbol-and-object). acredito que nesse caso aqui nem precisaria de tipagem ela é inferida